src/kartograf/filters/__init__.py

```diff
@@ -1,4 +1,4 @@
-from .element_change import (
```

Comment:
Because Kartograf claims a 1.0 we do indeed need to avoid the API break - it's being used in our tooling enough that we don't want to swap this out. We'll need to do a deprecation and aim for a change in 2.0.